target/linux/generic/config-5.15

```diff
@@ -3184,6 +3187,7 @@ CONFIG_LSM_MMAP_MIN_ADDR=65536
 # CONFIG_LTC2632 is not set
 # CONFIG_LTC2983 is not set
 # CONFIG_LTE_GDM724X is not set
+CONFIG_LTO_NONE=y
```

Comment:
Please remove this in the same commit from the target configurations:
```
target/linux/ipq806x/config-5.15:CONFIG_LTO_NONE=y
target/linux/mediatek/mt7622/config-5.15:CONFIG_LTO_NONE=y
target/linux/mediatek/mt7623/config-5.15:CONFIG_LTO_NONE=y
target/linux/ramips/mt7621/config-5.15:CONFIG_LTO_NONE=y
target/linux/sunxi/config-5.15:CONFIG_LTO_NONE=y
target/linux/x86/config-5.15:CONFIG_LTO_NONE=y
```

Reply:
Done